bundle-workflow/python/signing_workflow/signer.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+# Copyright OpenSearch Contributors.
+# SPDX-License-Identifier: Apache-2.0
+
+'''
+This class is responsible for signing an artifact using the OpenSearch-signer-client and verifying its signature.
+The signed artifacts will be found in the same location as the original artifacts.
+'''
+class Signer:
+    def __init__(self, git_repo):
+        self.git_repo = git_repo
+        self.git_repo.execute("./bootstrap","src")
```

Comment:
if these are executing within the repo, you should be able to execute ./src/bootstrap and not need to update the signature?